src/Features/Core/Portable/EditAndContinue/AbstractEditAndContinueAnalyzer.cs

```diff
@@ -3361,20 +3348,17 @@ private bool TryMapParameter(ValueTuple<SyntaxNode, int> parameterKey, IReadOnly
         }
 
         private void CalculateCapturedVariablesMaps(
-            SemanticModel oldModel,
             ImmutableArray<ISymbol> oldCaptures,
-            ISymbol oldMember,
             SyntaxNode oldMemberBody,
-            SemanticModel newModel,
             ImmutableArray<ISymbol> newCaptures,
             ISymbol newMember,
             SyntaxNode newMemberBody,
             BidirectionalMap<SyntaxNode> map,
-            [Out]ArrayBuilder<int> reverseCapturesMap,                 // {new capture index -> old capture index}
-            [Out]ArrayBuilder<SyntaxNode> newCapturesToClosureScopes,  // {new capture index -> new closure scope}
-            [Out]ArrayBuilder<SyntaxNode> oldCapturesToClosureScopes,  // {old capture index -> old closure scope}
-            [Out]List<RudeEditDiagnostic> diagnostics,
-            out bool hasErrors,
+            [Out]ArrayBuilder<int> reverseCapturesMap,
+            [Out]ArrayBuilder<SyntaxNode> newCapturesToClosureScopes,
+            [Out]ArrayBuilder<SyntaxNode> oldCapturesToClosureScopes,
+            [Out]List<RudeEditDiagnostic> diagnostics,                 // {new capture index -> old capture index}
+            out bool hasErrors,  // {new capture index -> new closure scope}
```

Comment:
❓ Did you use a refactoring for this? Looks like it messed up these comments.

Reply:
Ye,s used Change signature refactoring.
@dpoeschl

Reply:
I thought we had a bug for this already, but I can't seem to find it. @brettfo and I had a couple features with this problem but never got a fix put together.

Reply:
Created https://github.com/dotnet/roslyn/issues/33801

Reply:
@dpoeschl Thanks!